decker.yaml

```diff
@@ -25,6 +25,8 @@ static-resources:
 # resource-pack: exe:mario
 # resource-pack: exe:mono
 
+resource-pack: exe:tudo
```

Comment:
do not enforce tudo style for all decks in this project (even though it is the nicest style)

Reply:
Commented it out again